.goreleaser.yaml

```diff
@@ -45,6 +45,7 @@ checksum:
 
 changelog:
   sort: asc
+  use: github-native
```

Comment:
The use of `github-native` makes all other `changelog` settings ignored (like `sort`, `groups`, and `filters`).

Suggest removing them to not confuse about configs that are unused.

The side effect is that we don't get automatic grouping of PRs into these "Added"/"Fixed"/etc groups. Just wanted to point that out so you're also OK with that

Reply:
Maybe could try the `github` mode then (without `-native`)?

The docs however say it needs a PAT for that, which is less nice. But if I understand correctly then it would use PR names (and links) together while still doing grouping.

Reply:
thought about that option as compromise and tried to find in their example repositories how actually it would look. If we don't like it, we can change it. for the PAT part the docs mention it for `gitlab` but not for `github`

Reply:
> for the PAT part the docs mention it for `gitlab` but not for `github`

Ah nice catch. My lousy eyes fail me again